### ISSUE TITLE
stackedAreaChart: tooltip not show 

### ISSUE THREAD

**Issue by thienhv295** [novus/nvd3#1595]
I try to show tooltip on Statcked Area Chart. But seems it not working. Plz check link below:

[https://jsfiddle.net/pz7vcsjx/](https://jsfiddle.net/pz7vcsjx/)

Thanks.

**Comment by Kyeongan** [comment]
This seems a bug in 1.8.2-dev. It works with a stable version. You could include a stable version which is 1.8.1. now.

This is what you include. /* nvd3 version 1.8.2-dev (https://github.com/novus/nvd3) 2016-02-05 */

https://jsfiddle.net/newmkka/u523uokL/2/

**Comment by thienhv295** [comment]
@Kyeongan: Yes, I see. I know it work fine on 1.8.1 version. But 1.8.1 version has some bugs related to view box. So, I changed to 1.8.2 version (fixed view box bug).
